lib/fastlane/plugin/appcenter/actions/appcenter_upload_action.rb

```diff
@@ -230,7 +230,8 @@ def self.get_or_create_app(params)
         platforms = {
           Android: %w[Java React-Native Xamarin],
           iOS: %w[Objective-C-Swift React-Native Xamarin],
-          macOS: %w[Objective-C-Swift]
+          macOS: %w[Objective-C-Swift],
```

Comment:
How about having Unity for ios and android?
Should we also have tvOS?

Reply:
Not sure, how symbols and stuff works for Unity iOS..
For tvOS, there is no distribute support from portal.
Lets ignore them.. For now this should be fine.